api/pom.xml

```diff
@@ -19,21 +19,21 @@
 			<artifactId>fixture-factory</artifactId>
 			<version>3.1.0</version>
 		</dependency>
-		
+
 		<dependency>
```

Comment:
I really agree with you @ningosi and support you idea. I'll do everything I can do follow that approach.

Thanks